Docker.md

```diff
@@ -0,0 +1,38 @@
+# Run the Darkflame Server inside Docker
+
+## What you need
+
+- [Docker](https://docs.docker.com/get-docker/) (Docker Desktop or on Linux normal Docker)
+- [Docker Compose](https://docs.docker.com/compose/install/) (Included in Docker Desktop)
+- LEGO® Universe Client (packed or unpacked). Check the main [README](./README.md) for details on this.
+
+## Run server inside Docker
+
+1. Copy `.env.example` and save it as `.env` inside the root directory of this repository
+2. Edit the `.env` file and add your path to the root directory of your LEGO® Universe Client after `CLIENT_PATH=`
+3. Update other values in the `.env` file as needed (be sure to update passwords!)
+4. Run `docker compose up -d --build`
```

Comment:
I agree - it also means you can't just kill via ctrl+c (which iirc you can do otherwise?)

Reply:
Yes, you'd have to stop it with `docker-compose stop` or `down` I think.

Reply:
To be honest, I don't know what is better. With `docker compose up` (without -d) there is always the danger that the user don't understand he should not close the terminal window. With `docker compose up -d` there is the danger, that the user just forgets about the server and it always takes resources in the background. 

The advantage of `docker compose up -d` is, we could advice the user to just close Docker Desktop to stop the server. And if he wants to play, he just needs to start Docker Desktop again.

Reply:
I'm inclined to suggest that doing something without the user's knowledge is worse than it being stopped because of accidental action in this case.

TBH, the right answer is probably just list both - "Run docker-compose, or docker-compose -d to run in the background" (better written, of course)

Reply:
In the Windows.md I just told the user to use the play or stop buttons to stop and start the server. This gives the user some indication that the server is running, they don't have to remember the relevant commands, and they can still get logs from Docker Desktop.